package.json

```diff
@@ -16,7 +16,8 @@
     "lint-fix": "eslint --fix src/**/*.ts",
     "pretty": "prettier --write 'src/**/*.ts'",
     "precommit": "npm run pretty",
-    "prepush": "npm run lint"
+    "prepush": "npm run lint",
+    "copy-assets": "cp compose-network/network-node/data/config/evm-addresses-blocklist.csv build/state"
```

Comment:
We have to abandon this apporach unfortunately, because copiyng, pasting, moving is not allowed, simply because we are trying to avoid making the user using sudo command. Remember that this package is supposed to be installed globally.